### DISCUSSION TITLE
ADB backup and restore Music in another phone

### DISCUSSION THREAD

**Discussion by airon90** [z-huang/music#172] [Q&A] [answered]
I bought a new phone and I'd like to import 4k+ songs from Music in my new phone.
Is it possible to do, at least with adb?

**Comment by z-huang** [comment]
I'm not sure, but you can try. Maybe I'll bring the backup feature in the next version.

**Comment by airon90** [comment, marked as answer]
I did it: 
```
adb backup -f music.ab com.zionhuang.music
adb restore music.ab
```

**Reply by siggi1984** [reply]
it worked

**Reply by xmha97** [reply]
The backup on my phone is only 47 bytes.
[music.zip](https://github.com/z-huang/music/files/9617088/music.zip)

**Reply by z-huang** [reply]
Please wait for #57.

**Comment by z-huang** [comment]
Backup and restore is implemented.
